tests/db_config.py

```diff
@@ -1,23 +1,24 @@
+import os
 import peewee_async
 import peewee_asyncext
 
 
 PG_DEFAULTS = {
     'database': 'postgres',
     'host': '127.0.0.1',
-    'port': 5432,
+    'port': int(os.environ.get('POSTGRES_PORT', 5432)),
     'password': 'postgres',
     'user': 'postgres',
-    "connect_timeout": 30
+    'connect_timeout': 30
 }
 
 MYSQL_DEFAULTS = {
     'database': 'mysql',
     'host': '127.0.0.1',
-    'port': 3306,
+    'port': int(os.environ.get('MYSQL_PORT', 3306)),
```

Comment:
[nit](https://www.codelantis.com/blog/code-review-acronyms-lgtm-nit): [environs](https://github.com/sloria/environs) could be a bit convenient for environment variables

Reply:
@kalombos I suggest not to add an extra depencency here. This port customization is just for developer convinience in case port is already used.

For reading environemnt varialbes with strict validation if ever needed I'd strongly suggest using `pydantic ` + `pydantic-settings`.